# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1295034879 28800
#      Fri Jan 14 11:54:39 2011 -0800
# Node ID a10544c699f20bbc9afd265ad5d207d9568a56f8
# Parent  7ef6031cb35e7e2520008193c7d164591add17f9
Remove texas_lotto from Automake build system.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2011-01-14  Rik  <octave@nomad.inbox5.com>
+
+	* miscellaneous/module.mk: Remove texas_lotto.m from build system.
+
 2011-01-14  Rik  <octave@nomad.inbox5.com>
 
 	* miscellaneous/texas_lotto.m: Remove Easter Egg function.
 
 2011-01-14  John W. Eaton  <jwe@octave.org>
 
 	* Update copyright notices for 2011.
 
diff --git a/scripts/miscellaneous/module.mk b/scripts/miscellaneous/module.mk
--- a/scripts/miscellaneous/module.mk
+++ b/scripts/miscellaneous/module.mk
@@ -56,17 +56,16 @@ miscellaneous_FCN_FILES = \
   miscellaneous/setappdata.m \
   miscellaneous/setfield.m \
   miscellaneous/substruct.m \
   miscellaneous/swapbytes.m \
   miscellaneous/symvar.m \
   miscellaneous/tar.m \
   miscellaneous/tempdir.m \
   miscellaneous/tempname.m \
-  miscellaneous/texas_lotto.m \
   miscellaneous/unimplemented.m \
   miscellaneous/unix.m \
   miscellaneous/unpack.m \
   miscellaneous/untar.m \
   miscellaneous/unzip.m \
   miscellaneous/ver.m \
   miscellaneous/version.m \
   miscellaneous/warning_ids.m \
